python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -259,15 +259,27 @@ def test_upscale_downscale_delay():
 
     overload_requests = [100]
 
+    # scale up when there is 0 replica and current_handle_queued_queries > 0
+    new_num_replicas = policy.get_decision_num_replicas(
+        current_num_ongoing_requests=[],
+        curr_target_num_replicas=0,
+        current_handle_queued_queries=1,
+    )
+    assert new_num_replicas == 1
+
     # We should scale up only after enough consecutive scale-up decisions.
```

Comment:
Is this true for autoscaling in general? Do we wait for a certain number of scale up decisions?

Reply:
yes, it is true for scale down to 0.